libr/bin/p/bin_dex.c

```diff
@@ -339,7 +339,11 @@ static void dex_parse_debug_item(RBinFile *binfile, RBinDexObj *bin,
 		return;
 	}
 
+#ifdef _MSC_VER
```

Comment:
drop this ifdef, and use the heap for all platforms. but dont use malloc, we have macros like R_ANEW, and this fits better with calloc() than malloc().

dynamically sized stack arrays are dangerous, thats why we shuold avoid that pattern. thanks for spotting